src/plugins/kibana_react/public/exit_full_screen_button/exit_full_screen_button.tsx

```diff
@@ -54,8 +51,7 @@ class ExitFullScreenButtonUi extends PureComponent<ExitFullScreenButtonProps> {
           </p>
         </EuiScreenReaderOnly>
         <div className="dshExitFullScreenButton">
-          <KuiButton
-            type="hollow"
+          <button
             aria-label={i18n.translate(
               'kibana-react.exitFullScreenButton.exitFullScreenModeButtonAreaLabel',
```

Comment:
Also, a minor nit: can we rename this to `....AriaLabel`? (as opposed to `AreaLabel`?)

Reply:
Hmmm, but I am not sure if that's what we want here. I've looked through the project and I found some other instances of AreaLabels. Looks like you're right, as it doesn't make sense in this case, but can someone confirm it's AriaLabel what we expect here? 

![image](https://user-images.githubusercontent.com/4283304/69712487-303b7b00-1103-11ea-88ed-5dfb62309f8f.png)